python/ThirteenTeV/XXTo4J_M-1000_CTau-1000mm_TuneCUETP8M1_13TeV_pythia8_cff.py

```diff
@@ -0,0 +1,92 @@
+COM_ENERGY = 13000. # GeV
+MASS_POINT = 1000   # GeV
+CROSS_SECTION = 1 # pb
+CTAU_POINT = 1000 # mm
+
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CUEP8M1Settings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+    pythiaPylistVerbosity = cms.untracked.int32(0),
+    filterEfficiency = cms.untracked.double(1),
+    pythiaHepMCVerbosity = cms.untracked.bool(False),
+    comEnergy = cms.double(COM_ENERGY),
+    crossSection = cms.untracked.double(CROSS_SECTION),
+    maxEventsToPrint = cms.untracked.int32(0),
+    PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CUEP8M1SettingsBlock,
+        processParameters = cms.vstring(
+            "Higgs:useBSM = on",
+            "HiggsBSM:all = off",
+            "HiggsBSM:ffbar2A3H2 = on",
+            "35:m0 = %s" % MASS_POINT,
+            "36:m0 = %s" % MASS_POINT,
+            "35:mMin = 5.",
```

Comment:
The mMin and mMax settings look wrong
(mMin=5 and mMax=0)

Why are these needed at all?